src/Components/calculator.js

```diff
@@ -9,40 +9,34 @@ function NumButton(props) {
   return (<button type="button" className={classes} onClick={onClick}>{number}</button>);
 }
```

Comment:
[OPTIONAL] It's best practice in React to have all helper functions inside the main component. Nothing must be outside of the `Calculator()` function except for `imports` at the very top of the page and perhaps `PropTypes` at the bottom of the page if you have them.

All logic pertaining to a functional component must be above the `return` method.

Reply:
Thanks @KDlamini! I will work on it